src/PKSim.Core/Model/IndividualParameterSameFormulaOrValueForAllSpecies.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PKSim.Core.Model
+{
+   public class IndividualParameterSameFormulaOrValueForAllSpecies
+   {
+      public int ContainerId { get; set; }
+      public string ContainerPath { get; set; }
+      public string ParameterName { get; set; }
+      public bool IsSameFormula { get; set; }
+      public bool IsSameValue => !IsSameFormula;
```

Comment:
Well, it could Happen  that the same formula for different species results in the same value as well - but do we have any gain from this information?

Reply:
No idea;-). It's just that this seems maybe wrong lol. I'll see how I'm using it and I'll let you know ha ha

Reply:
This also depends on other individual parameters and is pretty hard to check

Reply:
The only case where it's easier to check: if the formula has no references (e.g. something like "24*60".
Then we would have same formula and same value.

Reply:
Forget it. Not sure if this makes sense. 
How should I use it. Get the object from path and check first if it exists same for all, if not est
Otherwise, what's the order of operations?

Reply:
Yes, I think so.
First check if parameter exists for all species (from that commonForAll-repo).
If yes: check if it has the same formula/value.